README.md

```diff
@@ -360,6 +361,60 @@ snet contract <ContractName> [--at AT] <functionName> PARAM1, PARAM2, ... [--tra
 
 ---
 
+```
+snet service init [--name NAME]
+                  [--model MODEL]
+                  [--organization ORGANIZATION]
+                  [--price PRICE]
+                  [--endpoint ENDPOINT]
+                  [--tags TAGS [TAG1, TAG2, ...]]
+                  [--description DESCRIPTION]
+                  [-y]
+```
+
+* Create a service.json file in the current directory either interactively or by passing command line arguments
+* `NAME`: this will be the name of the service in the Registry
+* `MODEL`: path relative to the service.json file where the service model .proto files are stored
+* `ORGANIZATION`: the service will be registered in this Organization
+* `PRICE`: job price for the service Agent
+* `ENDPOINT`: endpoint to call the service
+* `TAGS`: a list of tags for the service on the Registry
+* `DESCRIPTION`: a human-readable, high-level description of the service
+* `-y`: accept all defaults
+
+---
+
+```
+snet service publish <Network> [--no-register]
```

Comment:
If this is to be consistent with how the network command works, this should be

`snet service publish [NETWORK_NAME] [--no-register] [...`

or

`snet service publish [endpoint ENDPOINT] [--no-register] [...`

(same with the update command's use of `<Network>`)

Aside from how the documentation looks, do we currently support the use of an endpoint in these commands or does this only accept mainnet, kovan, rikeby, ropsten?

Reply:
I ended up doing it the way it's done in the network command; seemed more easy to document in a clear way